tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -43,111 +43,133 @@ def _generate_jitted_eigsh_lanczos(jax: types.ModuleType) -> Callable:
   @functools.partial(jax.jit, static_argnums=(3, 4, 5, 6, 7))
   def jax_lanczos(matvec: Callable, arguments: List, init: jax.ShapedArray,
                   ncv: int, neig: int, landelta: float, reortho: bool,
-                  precision: JaxPrecisionType):
+                  precision: JaxPrecisionType) -> Tuple[jax.ShapedArray, List]:
     """
-    Jitted lanczos routine.
+    Lanczos iteration for symmeric eigenvalue problems. If reortho = False,
+    the Krylov basis is constructed without explicit re-orthogonalization. 
+    In infinite precision, all Krylov vectors would be orthogonal. Due to 
+    finite precision arithmetic, orthogonality is usually quickly lost. 
+    For reortho=True, the Krylov basis is explicitly reorthogonalized.
     Args:
       matvec: A callable implementing the matrix-vector product of a
         linear operator.
       arguments: Arguments to `matvec` additional to an input vector.
         `matvec` will be called as `matvec(init, *args)`.
```

Comment:
Something weird is happening with the Markdown here

Reply:
There are a couple of places where this happends. I'll have to run through all files eventually to fix this